plugins/woocommerce/includes/rest-api/Controllers/Version3/class-wc-rest-product-variations-controller.php

```diff
@@ -535,7 +541,12 @@ public function get_item_schema() {
 					'readonly'    => true,
 				),
 				'sku'                   => array(
-					'description' => __( 'Unique identifier.', 'woocommerce' ),
+					'description' => __( 'SKU.', 'woocommerce' ),
```

Comment:
Should we still keep this as `'Unique identifier.'`?

Reply:
I prefer keeping it like this, to avoid confusion with the new field, though I don't think this description is a big deal. I'll update it to Stock Keeping Unit, is that OK?